index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1 class="header_name">Nicholas</h1>
```

Comment:
As per the project requirements, the logo should be a link instead of a heading, so when the visitors click on the logo, they are taken to the top of the page/home page. Also, there should only be one `h1` per page and it is reserved for the most important heading on the page. Don't you think, a logo is not the most important heading on a page? 😊